chain/consensus/mir/manager.go

```diff
@@ -42,7 +44,9 @@ const (
 
 	CheckpointDBKeyPrefix = "mir/checkpoints/"
 
-	ReconfigurationInterval = 2000 * time.Millisecond
+	ReconfigurationInterval   = 2000 * time.Millisecond
+	WaitForMembershipTimeout  = 60 * time.Second
```

Comment:
Why only 60 seconds? I feel we should make it larger or not even timeout for this and just loop until there's a sigterm. WDYT?

Reply:
Sorry, I did not see that comment. A user could rerun the system and try it again. We can definitely increase the timeout. Wouldn't it be better to report an error and finish the execution? Or do we require that?